docs/developer-docs/latest/setup-deployment-guides/configurations/required/databases.md

```diff
@@ -275,7 +280,249 @@ await pluginStore.set({
+    },
+    postgres: {
+      connection: {
+        connectionString: env('DATABASE_URL'),
+        host: env('DATABASE_HOST', 'localhost'),
+        port: env.int('DATABASE_PORT', 3306),
+        database: env('DATABASE_NAME', 'strapi'),
+        user: env('DATABASE_USERNAME', 'strapi'),
+        password: env('DATABASE_PASSWORD', 'strapi'),
+        ssl: env.bool('DATABASE_SSL', false) && {
+          key: env('DATABASE_SSL_KEY', undefined),
+          cert: env('DATABASE_SSL_CERT', undefined),
+          ca: env('DATABASE_SSL_CA', undefined),
+          capath: env('DATABASE_SSL_CAPATH', undefined),
+          cipher: env('DATABASE_SSL_CIPHER', undefined),
+          rejectUnauthorized: env.bool(
+            'DATABASE_SSL_REJECT_UNAUTHORIZED',
+            true
+          ),
+        },
+        schema: env('DATABASE_SCHEMA', 'public'),
+      },
+      pool: { min: env.int('DATABASE_POOL_MIN', 2), max: env.int('DATABASE_POOL_MAX', 10) },
+    },
+    sqlite: {
+      connection: {
+        filename: path.join(
+          __dirname,
+          '..',
+          env('DATABASE_FILENAME', 'data.db')
+        ),
+      },
+      useNullAsDefault: true,
+    },
+  };
+
+  return {
+    connection: {
+      client,
+      ...connections[client],
+      acquireConnectionTimeout: env.int('DATABASE_CONNECTION_TIMEOUT', 60000),
+    },
+  };
+};
+```
+
+</code-block>
+</code-group>
+
+The following are examples of the corresponding `.env` file `# Database` section for each of the possible databases:
+
+<code-group>
+<code-block title="MySQL">
+
+```bash
+
+# Database
+DATABASE_CLIENT=mysql
+DATABASE_HOST=127.0.0.1
+DATABASE_PORT=3306
+DATABASE_NAME=strapi
+DATABASE_USERNAME=strapi
+DATABASE_PASSWORD=strap1
+DATABASE_SSL=false
+```
+
+</code-block>
+
+<code-block title="PostgreSQL">
+
+```bash
+
+# Database
+DATABASE_CLIENT=postgres
+DATABASE_HOST=127.0.0.1
+DATABASE_PORT=5432
+DATABASE_NAME=strapi
+DATABASE_USERNAME=strapi
+DATABASE_PASSWORD=strapi
+DATABASE_SSL=false
+```
+
+</code-block>
+
+<code-block title="SQLite">
+
+```bash
+
+# Database
+DATABASE_CLIENT=sqlite
+DATABASE_FILENAME=.tmp/data.db
+```
+
+</code-block>
+</code-group>
+
+:::strapi
+If you started your project with a version prior to v4.6.2 you can convert your `database.js|database.ts` configuration file following this procedure:
+
+1. Update your application to v4.6.2 or a later version. See the [Updates and Migrations](/developer-docs/latest/update-migration-guides/migration-guides/migration-guides.md) section for any breaking changes that require specific fixes.
+2. Replace the contents of your `./config/database.js` or `./config/database.ts` file with the preceding JavaScript or TypeScript code.
+3. Add the environment variables from the preceding code example to your `.env` file.
+4. (_optional_) Add additional environment variables such as `DATABASE_URL` and the properties of the `ssl` object.
+5. Save the changes and restart your application.
+::::caution
```

Comment:
I think we should avoid embedding a callout in another callout. I'd say:
- Either the caution callout is really important: In that case, I'd suggest turning the whole "strapi" callout into a section. This section's title (h3) would be the title you would add to the strapi callout (see previous comment).
- Or we extract the sentence from the caution callout and highlight it (bold formatting?).

Since this page is getting longer and longer, I would go for the first solution. Can you think of another solution? 🤔

Reply:
Yes the first solution sounds good.